frontend/src/services/event/event.api.ts

```diff
@@ -88,6 +88,32 @@ export const getRsvps = async (
   }).then((res) => res.data);
 };
 
+export const getEventRSVPsForDownload = async (
+  id: string,
+  orderBy?: string,
+  orderDirection?: OrderDirection,
+  search?: string,
+  branchId?: string,
+  departmentId?: string,
+  roleId?: string,
+  going?: string,
+): Promise<{ data: unknown; headers: AxiosResponseHeaders }> => {
+  return API.get(`event/${id}/rsvp/download`, {
+    params: {
+      orderBy,
+      orderDirection,
+      search,
+      branchId,
+      departmentId,
+      roleId,
+      going: going === undefined ? going : going === RsvpEnum.GOING,
```

Comment:
De ce facem asta? Daca nu avem filtru de Going n-ar trebui sa le intoarcem pe toate both going and not going?

Reply:
Daca nu este filtru atunci se trimite `undefined`. Daca este, se trimite comparatia cu `RsvpEnum.GOING`, deci `true` sau `false`.